File: backend/schemas/sales_coach.py
"""Response schema for GET /api/v1/companies/{id}/sales-coach (roadmap
Phase 4, item 10 - "What Would You Do?").
"""

from typing import Optional

from pydantic import BaseModel


class SalesCoachRecommendation(BaseModel):
    who_to_contact: Optional[str] = None
    best_talking_points: list[str] = []
    best_timing: Optional[str] = None
    risks: list[str] = []
    suggested_sequence: list[str] = []
    why: Optional[str] = None

File: frontend/react/src/types/salesCoach.ts
// Mirrors backend/schemas/sales_coach.py's SalesCoachRecommendation
// exactly (roadmap Phase 4, item 10 - "What Would You Do?").

export interface SalesCoachRecommendation {
  who_to_contact: string | null;
  best_talking_points: string[];
  best_timing: string | null;
  risks: string[];
  suggested_sequence: string[];
  why: string | null;
}
